public/docs/ts/latest/guide/router.jade

```diff
@@ -1339,6 +1582,28 @@ a#can-activate-guard
    app/login.routing.ts
   `)
 
+h3#can-activate-child-guard <i>CanActivateChild</i>: guarding child routes
+:marked
+  As we learned about guarding routes with `CanActivate`, we can also protect child routes with the `CanActivateChild`
+  guard. The `CanActivateChild` guard works similarly to the `CanActivate` guard, but the difference is its run _before_
+  each child route is activated. We protected our admin feature module from unauthorized access, but we could also
+  protect child routes within our feature module.
+
+  Let's extend our `AuthGuard` to protect when navigating between our `admin` routes. First we'll open our
+  `auth-guard.service.ts` and add `CanActivateChild` interface to our imported tokens from the router package.
+
+  Next, we'll implement the `canActivateChild` method with takes the same arguments as the `canActivate` method,
+  an `ActivatedRouteSnapshot` and `RouterStateSnapshot`. The `canActivateChild` behaves the same way the other
+  guards do, returning an `Observable<boolean>` or `Promise<boolean>` for async checks and `boolean` for sync checks.
+
++makeExcerpt('app/auth-guard.service.3.ts (excerpt)', 'can-activate-child')
```

Comment:
I agree about removing the return type. We are explaining the types of turns it expects. It would make the signatures in the examples shorter also.